FluentUI.Demo/src/main/res/drawable/fluent_logo.xml

```diff
@@ -0,0 +1,50 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Good pratice to have the xml version tag. Let us maintain parity across the xml files